### ISSUE TITLE
A question about backword process

### ISSUE THREAD

**Issue by Zbozhou** [liyingyanUCAS/WoTE#4]
Thanks for your excellent work, I am wondering why you set `loss.backward(retain_graph=True)` [here](https://github.com/liyingyanUCAS/WoTE/blob/main/navsim/planning/training/agent_lightning_module.py#L60).

**Comment by Zbozhou** [comment]
Thanks for your reply.
